src/scenes/ProgressReports/Detail/WorkflowCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Card, CardActions, StepperProps, Typography } from '@mui/material';
+import { ProgressReportStatus as Status } from '~/api/schema/schema.graphql';
+import { RelativeDateTime } from '~/components/Formatters';
+import { ButtonLink } from '~/components/Routing';
+import { ProgressReportDetailFragment } from './ProgressReportDetail.graphql';
+import { StatusStepper } from './StatusStepper';
+
+type WorkflowCardProps = {
+  report?: ProgressReportDetailFragment;
+} & StepperProps;
+
+export const WorkflowCard = ({ report, ...rest }: WorkflowCardProps) => {
+  const currentStatus = report?.status.value as Status | undefined;
+
+  const currentWorkflowEvent = report?.workflowEvents.at(
+    report.workflowEvents.length - 1
+  );
+  return (
+    <Card {...rest}>
+      <Typography variant="h3" sx={{ pl: 2, pt: 2 }}>
+        Status
+      </Typography>
+      <StatusStepper loading={!report} current={currentStatus} />
```

Comment:
Inline currentStatus here, also the typecast is not needed.
```suggestion
      <StatusStepper loading={!report} current={report?.status.value} />
```